<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlotStyleEditingWidget</class>
 <widget class="QWidget" name="PlotStyleEditingWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>599</width>
    <height>389</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>8</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Axis rect</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Axis rect</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="AxisRectEditingWidget" name="axisRectEditingWidget" native="true"/>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>0</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Margins</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <widget class="MarginsEditingWidget" name="marginsEditingWidget" native="true"/>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>0</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_8">
      <attribute name="title">
       <string>Axes pref.</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_18">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QGroupBox" name="groupBox_13">
           <property name="title">
            <string>X axis</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_16">
            <item>
             <widget class="AxisPrefEditingWidget" name="xAxisPrefWidget" native="true"/>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_14">
           <property name="title">
            <string>Y axis</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_17">
            <item>
             <widget class="AxisPrefEditingWidget" name="yAxisPrefWidget" native="true"/>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_7">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Grid</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Grid</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="GridEditingWidget" name="xGridEditingWidget" native="true"/>
          </item>
          <item>
           <widget class="GridEditingWidget" name="yGridEditingWidget" native="true"/>
          </item>
          <item>
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>0</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>Subgid</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="GridEditingWidget" name="xSubgridEditingWidget" native="true"/>
          </item>
          <item>
           <widget class="GridEditingWidget" name="ySubgridEditingWidget" native="true"/>
          </item>
          <item>
           <spacer name="verticalSpacer_4">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>0</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Ticks</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QGroupBox" name="groupBox_5">
         <property name="title">
          <string>Tick</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="TickEditingWidget" name="tickEditingWidget" native="true"/>
          </item>
          <item>
           <spacer name="verticalSpacer_5">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>0</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_6">
         <property name="title">
          <string>Subtick</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="TickEditingWidget" name="subtickEditingWidget" native="true"/>
          </item>
          <item>
           <spacer name="verticalSpacer_6">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>0</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_5">
      <attribute name="title">
       <string>Legend</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_9">
       <item>
        <widget class="LegendEditingWidget" name="legendEditingWidget" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_7">
      <attribute name="title">
       <string>Labels</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="0">
        <widget class="QGroupBox" name="groupBox_10">
         <property name="title">
          <string>X tick label</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_15">
          <item>
           <widget class="LabelPrefsEditingWidget" name="xTickLabelEditingWidget" native="true"/>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QGroupBox" name="groupBox_9">
         <property name="title">
          <string>Y label</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_12">
          <item>
           <widget class="LabelPrefsEditingWidget" name="yLabelEditingWidget" native="true"/>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox_8">
         <property name="title">
          <string>X label</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_11">
          <item>
           <widget class="LabelPrefsEditingWidget" name="xLabelEditingWidget" native="true"/>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QGroupBox" name="groupBox_11">
         <property name="title">
          <string>Y tick label</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_13">
          <item>
           <widget class="LabelPrefsEditingWidget" name="yTickLabelEditingWidget" native="true"/>
          </item>
         </layout>
        </widget>
       </item>
       <item row="3" column="0">
        <spacer name="verticalSpacer_12">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="2">
        <widget class="QGroupBox" name="groupBox_12">
         <property name="title">
          <string>Title</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_14">
          <item>
           <widget class="LabelPrefsEditingWidget" name="titleEditingWidget" native="true"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_6">
      <attribute name="title">
       <string>Colors</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_10">
       <item>
        <widget class="ColorSetEditingWidget" name="colorPrefEditingWidget" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>Events</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QGroupBox" name="groupBox_7">
         <property name="title">
          <string>Events</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_19">
          <item>
           <widget class="LevelPrefsEditingWidget" name="eventEditingWidget" native="true"/>
          </item>
          <item>
           <spacer name="verticalSpacer_8">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_15">
         <property name="title">
          <string>Horizontal levels</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_20">
          <item>
           <widget class="LevelPrefsEditingWidget" name="horLevelEditingWidget" native="true"/>
          </item>
          <item>
           <spacer name="verticalSpacer_9">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_9">
      <attribute name="title">
       <string>Arrowed text</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_21">
       <item>
        <widget class="QGroupBox" name="groupBox_16">
         <property name="title">
          <string>Arrowed text properties</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <item>
           <widget class="ArrowedTextPrefsEditingWidget" name="arrowedTextEditingWidget" native="true"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_10">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ColorSetEditingWidget</class>
   <extends>QWidget</extends>
   <header>commonWidgets/editingWidgets/colorseteditingwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LabelPrefsEditingWidget</class>
   <extends>QWidget</extends>
   <header>commonWidgets/editingWidgets/labelprefseditingwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GridEditingWidget</class>
   <extends>QWidget</extends>
   <header>commonWidgets/editingWidgets/grideditingwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>AxisRectEditingWidget</class>
   <extends>QWidget</extends>
   <header>commonWidgets/editingWidgets/axisrecteditingwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TickEditingWidget</class>
   <extends>QWidget</extends>
   <header>commonWidgets/editingWidgets/tickeditingwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>AxisPrefEditingWidget</class>
   <extends>QWidget</extends>
   <header>commonWidgets/editingWidgets/axisprefeditingwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MarginsEditingWidget</class>
   <extends>QWidget</extends>
   <header>commonWidgets/editingWidgets/marginseditingwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LegendEditingWidget</class>
   <extends>QWidget</extends>
   <header>commonWidgets/editingWidgets/legendeditingwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LevelPrefsEditingWidget</class>
   <extends>QWidget</extends>
   <header>commonWidgets/editingWidgets/levelprefseditingwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ArrowedTextPrefsEditingWidget</class>
   <extends>QWidget</extends>
   <header>commonWidgets/editingWidgets/arrowedtextprefseditingwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
